docs/common/error-utilities.ts

```diff
@@ -152,7 +152,8 @@ const RENAMED_PAGES: Record<string, string> = {
   // Lots of old links pointing to guides when they have moved elsewhere
   '/guides/configuration/': '/workflow/configuration/',
   '/guides/expokit/': '/expokit/overview/',
-  '/guides/publishing/': '/workflow/publishing/',
+  '/guides/publishing/': '/archive/classic-updates/publishing/',
+  '/workflow/publishing/': '/archive/classic-updates/publishing/',
```

Comment:
I'm not sure I understand the difference between `/archive/some-link/` and `/archived/some-link/`? I guess, they both are for archiving. Is it because the links under `archived` are still visible but the links under `archive` are only visible when redirected?